instat/dlgOneWayFrequencies.vb

```diff
@@ -21,8 +21,7 @@ Public Class dlgOneWayFrequencies
     Private bReset As Boolean = True
     Private bResetSubdialog As Boolean = False
     Private clsSjTab As New RFunction
-    Private clsSjPlot, clsPlotGrid, clsSelect As New RFunction
-    Private clsTableBaseOperator, clsGraphBaseOperator As New ROperator
+    Private clsSjPlot, clsPlotGrid As New RFunction

```

Comment:
For the one way frequencies, I do not have objection to the use of `sjtab` function in place of dplyr pipes for this case. I essentially find both doing the same thing. I think for the three way frequencies, that is where it will be unavoidable. Though Roger was for using the pipe as he stated in  #2938